src/components/ProcessingCard/LinkButton.tsx

```diff
@@ -1,19 +1,14 @@
 import { LinkText } from 'components/ui/Text'
+import { scrollToHashElement } from 'hooks/useScrollToAnchor'
 import Button from 'components/ui/Button'
 
 interface LinkButtonProps {
   url: string
   internal?: boolean
   approved?: boolean
-  onClick?: () => void
 }
 
-export default function ({
-  url,
-  internal,
-  approved,
-  onClick,
-}: LinkButtonProps) {
+export default function ({ url, internal, approved }: LinkButtonProps) {
```

Comment:
If we'll use this component somewhere else than status, then we can add `onClick`, right now it only used in the top status block